client/asset/btc/btc.go

```diff
@@ -2191,7 +2219,7 @@ func (btc *ExchangeWallet) spendableUTXOs(confs uint32) ([]*compositeUTXO, map[o
 			// unlocked something or even restarted the wallet software.
 			pt := newOutPoint(txHash, txout.Vout)
 			if btc.fundingCoins[pt] != nil {
-				btc.log.Warnf("Known order-funding coin %v returned by listunspent!", pt)
+				btc.log.Warnf("Known order-funding coin %s returned by listunspent!", pt)
```

Comment:
This line is why I changed the `outPoint`'s Stringer to value receiver. https://github.com/decred/dcrdex/pull/865/files/812da1ebbf8eb03471bb3d80328dd177f8eac16c#diff-1b07e494f4bdd99b2a617ac110fbadc081e5b6ae1abf23f64032bbee5ebebef1R177
It was just showing hex or something definitely not decoded.